simplq/src/components/pages/Status/QueueDetails.jsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
+import moment from 'moment';
+import { CircularProgress } from '@material-ui/core';
 import Header from '../../common/Header';
 import styles from '../../../styles/statusPage.module.scss';
+import * as QueueService from '../../../services/queue';
+import { handleApiErrors } from '../../ErrorHandler';
 
 export default (props) => {
   // https://dabblet.com/gist/1506530 --> checkbox hack
+  const [details, setDetails] = useState();
+  QueueService.getStatus(props.queueId).then(setDetails).catch(handleApiErrors);
+  if (!details) {
+    return <CircularProgress />;
+  }
+  const timeStamp = moment(details.queueCreationTimestamp);
+  const creationTime = `${timeStamp.format('LT')} ${timeStamp.format('ll')}`;
```

Comment:
Does this take care of timezone difference between server and the user? For example, say server returns UTC time, and client is in India, does this display the converted Indian time?

Reply:
Right now, it just converts the time. Let me see if there's a way to fgure out the timezone dynamically.

Reply:
It's stored as IST right now in the backend it seems. Can we have it stored as UTC? That way, using solutions like this would make things easy and intuitive 
https://stackoverflow.com/questions/1091372/getting-the-clients-timezone-offset-in-javascript
https://momentjs.com/docs/#/manipulating/utc-offset/

Reply:
Yo. I'll do this weekend. I've raised a BE issue here https://github.com/SimplQ/simplQ-backend/issues/79. Can you also make a FE issue, and let's move on.

Reply:
Backend is giving UTC

Reply:
Really? It shows the right time in IST on the frontend, so I assumed it was storing time in IST. Let me see how moment figures out the timezone.

Reply:
Ah, so backend sends you the UTC time and the timezone is also specified. Moment knows your device's timezone, and might be doing the TZ conversion. That's my guess

Reply:
How does the backend figure out the right timezone if it's not stored? Does it find that out from where the request is sent from? The creationTimeStamp looks like this right now: 2020-08-10T20:23:08.745+0000. I've left a TODO in the code to look at this later. The other stuff is resolved now.

Reply:
Backend doesn't know the timezone of the user. Sorry for the confusion. What I had meant was that backend stores the current UTC time. The timezone in the datetime string from backend is "+0000", meaning it's UTC/GMT. For example, backend will send the UI, `2020-08-11T20:22:13.118+0000`

Now moment knows you are in IST timezone, (+0530) and it will add the offset and display the time in IST.

Reply:
Got it. This is probably right.